pallets/confidential-asset/src/lib.rs

```diff
@@ -423,8 +464,8 @@ decl_error! {
         /// The user is not authorized.
         Unauthorized,
 
-        /// Only confidential assets' total supply can change after registering the asset.
-        NonConfidentialAssetTotalSupplyCannotBeSet,
+        /// The provided asset is not among the set of valid asset ids.
+        ConfidentialAssetIsInvalid,
```

Comment:
Maybe `UnknownConfidentialAsset` would be more descriptive here.

Reply:
Sounds good. Will change it tomorrow.